jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MultiPartByteRanges.java

```diff
@@ -181,26 +212,38 @@ protected boolean isReadComplete(long read)
      */
     public static class Part extends MultiPart.Part
     {
-        private final Path path;
+        private final Resource resource;
```

Comment:
Please change the class javadocs to not use "file".

Reply:
Done.